### ISSUE TITLE
HTTP HEADERS ISSUE

### ISSUE THREAD

**Issue by SilasRodrigues19** [cijiugechu/astro-link-preview#15]
Hey

I saw that your lib takes all the links on the page to be able to create the link-preview

```mjs
const anchorElements = document.querySelectorAll('a')
```

It happens that when there is a link that does not have `HTTP Headers`, the application breaks. For example:

Native email links that end up crashing the application when hovering element

```html
<a href="mailto:mail@domain.com">Email</a>
```

Error that is thrown: 
  
```sh
  Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
  code: 'ERR_HTTP_HEADERS_SENT'
```

Would there be any way to validate whether the link starts with `http` or `https`? Something like that:

```mjs
href && (href.startsWith('http://') || href.startsWith('https://'));
```

**Comment by cijiugechu** [comment]
Fixed in [4e983ab](https://github.com/cijiugechu/astro-link-preview/commit/4e983abdc4996ee2b25a1a5e5f4578809942dfb1), you can update to `0.3.4` to resolve this issue.

**Comment by SilasRodrigues19** [comment]
Thanks @cijiugechu
